src/text-expander-element.ts

```diff
@@ -32,6 +32,8 @@ class TextExpander {
   onmousedown: (event: Event) => void
   combobox: Combobox | null
   match: Match | null
+  matchInProgress: boolean
+  currentMatchIndex: number
```

Comment:
How about we use `match: Match | null` for that. It seems that it already provides desired information.
By checking the code we would have to:
- adapt `findMatch()` to use `this.match` to read captured state
- update `deactivate()` to nullify `this.match`

Reply:
Good suggestion. Removed these fields and reused the `match` itself.